### PR TITLE
ClientConf digest during build

### PR DESCRIPTION
**Pull request by jmwample**
Added a script to automatically digest the conjure `assets/ClientConf` file and insert it into the psiphon core during build. This allows the build process to use the ClientConf associated with the specific branch that is pushed, making automated build for ClientConf testing easier.

### CODE REVIEW

File: test_scripts/digest_cc.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+TMP_SEDFILE="build/embedded_config.sed"
+
+# Digest the clienConf to hex
+EMBEDDED_CC=$(hexdump -ve '"\\\\\\\\\x" 1/1 "%.2x"' $1);
+
+# Create search and replace rule
```
Comment (sergeyfrolov) [inline test_scripts/digest_cc.sh]: Something along the lines of "Store search-and-replace rule in a script-file" would be clearer

File: test_scripts/digest_cc.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+TMP_SEDFILE="build/embedded_config.sed"
```
Comment (sergeyfrolov) [inline test_scripts/digest_cc.sh]: Please add a comment describing what this script does, and what input arguments are. It seems to me that the script is pretty Psiphon-specific, and, if so, we should mention that in the description comment.

What would this script do, if the script does not provide the required 2 arguments? Would be better to exit early on that.

File: test_scripts/digest_cc.sh (hunk 1 above)
Comment (jmwample) [reply test_scripts/digest_cc.sh]: Added change

File: test_scripts/digest_cc.sh (hunk 2 above)
Comment (jmwample) [reply test_scripts/digest_cc.sh]: added description and noted that this is psiphon specific.

**Pull request merged**